wallet-sidebar.js

```diff
@@ -41,6 +41,11 @@ const sidebar = {
               type: "doc",
               label: "Add a network",
               id: "how-to/manage-networks/add-network"
+            },
+            {
+              type: "doc",
+              label: "Interact with multiple networks",
```

Comment:
Is there a way to make the label more clear that we're connecting multiple blockchain networks in MetaMask simultaneously.

Reply:
Updated the title to "Interact with multiple networks simultaneously." This looks fine to me, but lmk if you think it's too long.